### PR TITLE
Change dialog for interactive assembler & patcher

### PR DESCRIPTION
**Pull request by quangnh89**
- Add accelerator prefix characters.
 - Set the keyboard focus to 'Assembly' text control.

### CODE REVIEW

File: keypatch.py [hunk 1]
```diff
@@ -759,16 +763,16 @@ def __init__(self, kp_asm, address, assembly=None, opts=0):
...
-             <-   Fixup :{c_raw_assembly}>
-             <-   Encode:{c_encoding}>
-             <-   Size  :{c_encoding_len}>
-            <Padding extra bytes with NOPs:{c_opt_padding}>{c_opt_chk}>
+            <~S~yntax     :{c_syntax}>
+            <A~d~dress    :{c_addr}>
+            <~O~riginal   :{c_orig_assembly}>
+             <-   E~n~code:{c_orig_encoding}>
+             <-   S~i~ze  :{c_orig_len}>
+            <~A~ssembly   :{c_assembly}>
```
Comment (aquynh) [inline keypatch.py]: except the control Assembly & Syntax, other controls (such as Address, Encode, Size, etc) do not accept input. so i think there is no need to setup the keys for them.

**Comment by aquynh** [comment]
this looks good to me, except issue in the inline comment.

thanks.

**Comment by quangnh89** [comment]
Thank you for reviewing. I have removed unnecessary keys .

File: keypatch.py [hunk 2]
```diff
@@ -843,14 +849,14 @@ def __init__(self, kp_asm, address, assembly=None):
...
-             <-   Encode:{c_encoding}>
-             <-   Size  :{c_encoding_len}>
+            <A~r~ch       :{c_arch}>
+            <E~n~dian     :{c_endian}>
+            <~S~yntax     :{c_syntax}>
+            <A~d~dress    :{c_addr}>
+            <~A~ssembly   :{c_assembly}>
+             <-   ~F~ixup :{c_raw_assembly}>
+             <-   ~E~ncode:{c_encoding}>
+             <-   S~i~ze  :{c_encoding_len}>
```
Comment (aquynh) [inline keypatch.py]: "Fixup" & "Size" are also read-only controls, so they should be fixed as well

**Comment by quangnh89** [comment]
accelerator keys from "Fixup" & "Size" are removed. :)

**Comment by aquynh** [comment]
i removed the accelerator key for "Encode" field because it is not really useful, and it makes the control unaligned with others. merged, thanks!

**Comment by aquynh** [comment]
we have some simple features in the TODO list at https://github.com/keystone-engine/keypatch/wiki. if you are bored & out of ideas, feel free to pick up something from the list, thanks!

**Pull request merged**